### ISSUE TITLE
FormRequest is giving a 500 Call to a member function make() on null

### ISSUE THREAD

**Issue by matyre73** [dingo/api#1573]
Q   |   A
--- | ----
Bug | yes
Framework  | Laravel
Framework version | 5.6.15
Package version  |  dev-master
PHP version      |  7.1

#### Actual Behaviour

I have set up the Form Request as below code 
```php
namespace App\Http\Requests;

use Config;
use Dingo\Api\Http\FormRequest;

class SignUpRequest extends FormRequest
{
    /**
     * Determine if the user is authorized to make this request.
     *
     * @return bool
     */
    public function authorize()
    {
        return true;
    }

    /**
     * Get the validation rules that apply to the request.
     *
     * @return array
     */
    public function rules()
    {
        return Config::get('boilerplate.sign_up.validation_rules');
    }
}
```

But I get the following error output.
```javascript
[
      "message" => "Call to a member function make() on null"
      "status_code" => 500
      "debug" => array:4 [
        "line" => 98
        "file" => "/var/www/vendor/dingo/api/src/Http/FormRequest.php"
        "class" => "Symfony\Component\Debug\Exception\FatalThrowableError"
       ]
]
```

#### Expected Behaviour

I expect to be no errors at all, however it is not even getting to checking the errors. Any help would be great.

**Comment by dliebel** [comment]
Hello, I have not found a clean solution but with this I partially solve it

https://github.com/dingo/api/pull/1542/files/674d3abc018f843ca7d18fac707c17806f816c6d

sorry for my english

**Comment by dliebel** [comment]
Hello, I am solved by upgrading to 2.4.
I am using the FormRequest of

use Illuminate\Foundation\Http\FormRequest;

I use version 5.8 of Laravel and Dingo 2.4+

`<?php



namespace App\Http\Requests;
use Illuminate\Foundation\Http\FormRequest;
class TokenAuthRequest extends FormRequest
{
    /**
     * Determine if the user is authorized to make this request.
     *
     * @return bool
     */
    public function authorize()
    {
        return true;
    }
    /**
     * Get the validation rules that apply to the request.
     *
     * @return array
     */
    public function rules()
    {
        return \App\User::$rules;
    }
}`
